cla_public/templates/base.html

```diff
@@ -84,6 +118,36 @@
 
 {% block javascripts %}
   <script>
+
+    var currentLocation = window.location.href;
+    if (currentLocation.indexOf("n43n3") > 0 || currentLocation.indexOf("n43n14/n97") > 0 || currentLocation.indexOf("n88") > 0 || currentLocation.indexOf("n149") > 0 || currentLocation.indexOf("n86") > 0) {
+      $(".laa-flee-button-container").addClass("laa-fee-button-enabled");
+    }
+
+    function flee() {
+      $("body").hide();
+      {#
```

Comment:
Do these comments get rendered on the page?

Reply:
No

Reply:
Can we remove the commented out code then if it's not being used

Reply:
Done